extensions/wikia/UserChangesHistory/UserChangesHistory.class.php

```diff
@@ -79,13 +79,6 @@ static public function LoginHistoryHook( $from, $user, $type = false ) {
 						// @see PLATFORM-1309
 						$dbw_specials = wfGetDB( DB_MASTER, array(), $wgSpecialsDB ) ;
 
-						$dbw_specials->insert(
-							"user_login_history",
-							$params,
-							__METHOD__,
-							array('IGNORE')
-						);
-
```

Comment:
:+1: 

The same needs to be made in the Perl backend repo that handles Scribe events.